graph.js

```diff
@@ -8,8 +8,17 @@ var Graph = (function (undefined) {
 		return keys;
 	}
 
+	function isFunction(functionToCheck) {
+		var getType = {};
+		return functionToCheck && getType.toString.call(functionToCheck) === '[object Function]';
+	}
+
+	function getWeight(value){
+		return isFunction (value) ? value() : parseFloat (value);
```

Comment:
Is there a reason why you're not just doing something like this here?
```javascript
return typeof value === 'function' ? value() : parseFloat(value)?
```
The `isFunction` method feels a little redundant, unless I'm missing something.